_rules/video-audio-described-1ea59c.md

```diff
@@ -71,7 +71,7 @@ A video element with an audio description.
 <figure id="ozplayer-1-container" class="ozplayer-container">
   <div data-controls="stack" class="ozplayer" id="ozplayer-1">
     <video controls="controls" preload="none">
-      <source src="../test-assets/rabbit-video.mp4" type="video/mp4"></source>
+      <source src="../test-assets/rabbit-video.mp4" type="video/mp4">
```

Comment:
Is `<source>` selv-closing or not? (`<source>` or `<source />`?) because both are used in this file.

Reply:
I fixed up the `/>` used in `source`. Unfortunately `htmlhint` - https://htmlhint.io/ only catches `</someTag>` on tags that do not allow closing tags but not `/>`.

But I would think having this test is better for many other reasons.